src/ui/components/Input/index.tsx

```diff
@@ -5,7 +5,7 @@ import "./input.scss";
 
 export interface InputProps extends InputHTMLAttributes<HTMLInputElement> {
   label: string;
-  icon?: ReactElement;
+  icon?: ReactElement[];
```

Comment:
It's not obvious why icon must be an array. Maybe it's better to rename to `endAdornment` and use only one element.

Reply:
> It's not obvious why icon must be an array

The intention is to support multiple icons in the Input icon prop as you can see from here: https://github.com/CryptKeeperZK/crypt-keeper-extension/blob/feature/show-hide-password-onboarding/src/ui/pages/Onboarding/Onboarding.tsx#L31-L62

> Maybe it's better to rename to endAdornment and use only one element.

This one is not clear to me, could you elaborate more please?

Reply:
@iskdrews you can take a look at mui naming. I propose to use the same API as mui has.
https://mui.com/material-ui/react-text-field/#input-adornments.
Because it's not obvious why not plural property `icon` expects array :)
You can use wrapper to keep single react element and rename it to `endAdornment` so when we migrate to mui it won't be a problem for us.